tests/phpunit/CRM/Dedupe/MergerTest.php

```diff
@@ -894,6 +922,7 @@ public function setupMatchData() {
     foreach ($fixtures as $fixture) {
       $contactID = $this->individualCreate($fixture);
       $this->contacts[] = array_merge($fixture, ['id' => $contactID]);
+      sleep(20);
```

Comment:
Can we get away with 5 or 10?

Reply:
Yeh i guess i was wanting something 'meaningful'